performance/falloutPerformanceTest.yaml

```diff
@@ -0,0 +1,183 @@
+          java.version: "1.8_151"
+          product.type: dse
+          product.install.type: tarball
+          product.version: 6.7-dev
+          enable.graph: false
+          json.topology: |
+            {
+               "cluster":
+                {
+                  "snitch":"GossipingPropertyFileSnitch",
+                  "nodes":
+                  {
+                    "0":{"datacenter":"datacenter1", "rack":"rack1", "node_type":"Cassandra", "seed":"True"},
+                    "1":{"datacenter":"datacenter1", "rack":"rack1", "node_type":"Cassandra", "seed":"True"},
+                    "2":{"datacenter":"datacenter1", "rack":"rack1", "node_type":"Cassandra"}
+                  }
+                }
+            }
+      - name: ctool_monitoring
+        properties:
+          components: os,jvm, dse-db, cassandra-all
+  observer:
+    node.count: 1
+    provisioner:
+      name: ctool
+      properties:
+        cloud.tenant: performance
+        cloud.instance.type: m1.xlarge
+        mark_for_reuse: true
+    configuration_manager:
+      - name: ctool_monitoring
+        properties:
+          graphite.create_server: true
+  clients:
+    - name: quickstartapp
+      node.count: 1
+      provisioner:
+        name: ctool
+        properties:
+          cloud.provider: openstack
+          cloud.tenant: performance
+          cloud.instance.type: ms1.small
+          mark_for_reuse: true
+      configuration_manager:
+        - name: ctool
+          properties:
+            install.maven: true
+            java.version: openjdk8
+        - name: ctool_monitoring
+          properties:
+            components: os
+
+    - name: http-client
+      node.count: 1
+      provisioner:
+        name: ctool
+        properties:
+          cloud.provider: openstack
+          cloud.tenant: performance
+          cloud.instance.type: ms1.small
+          mark_for_reuse: true
+      configuration_manager:
+        - name: ctool
+          properties:
+            install.maven: true
+            java.version: openjdk8
+        - name: ctool_monitoring
+          properties:
+            components: os
+
+
+workload:
+  phases:
+    - setup-dse-schema:
+        module: cqlsh
+        properties:
+          num.nodes: 1
+          command: >
+            CREATE KEYSPACE IF NOT EXISTS k1 WITH replication = {'class':'SimpleStrategy', 'replication_factor':2};
+            CREATE TABLE IF NOT EXISTS k1.fruit(store_id text, name text, description text, PRIMARY KEY((store_id), name));
+    - clone-and-build-quickstart-app:
+        module: bash
+        properties:
+          target.group: quickstartapp
+          export_output: false
+          timeout: 1 hours
+          script: |
+            cd ${FALLOUT_SCRATCH_DIR}
+            sudo apt update --assume-yes
+            sudo apt-get install lftp --assume-yes
+            sudo apt-get install zlib1g-dev libxmu-dev libxmu-headers freeglut3-dev libxext-dev libxi-dev --assume-yes
```

Comment:
Wow, why do we need all of this?

Reply:
those are needed for native graal compilation